code/java/index1.java

```diff
@@ -0,0 +1,103 @@
+package code.java;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.xapian.Document;
+import org.xapian.Stem;
+import org.xapian.TermGenerator;
+import org.xapian.WritableDatabase;
+import org.xapian.XapianConstants;
+import org.xapian.XapianJNI;
+
+public class index1 {
+    
+    // Command line args - dbpath datapath
+    public static void main(String[] args)
+    {
+        if(args.length < 2)
+        {
+            System.out.println("Insufficient number of arguments (should be dbpath datapath)");
```

Comment:
No, must be `DATAPATH DBPATH` as per the other examples. (Because these are all run automatically, they have to exactly the same arguments, in the same order.)